.circleci/config.yml

```diff
@@ -8,60 +8,54 @@ parameters:
     type: boolean
     description: Whether the pipeline was triggered through the CircleCi API (https://circleci.com/docs/api/v2/?shell#trigger-a-new-pipeline).
     default: false
-  build_config:
-    type: enum
-    description: Which build config to use and where to deploy to.
-    enum: ["integreat", "integreat-test-cms"]
-    default: "integreat-test-cms"
+  production_delivery:
+    type: boolean
+    description: Whether to deliver to production (integreat.app and malteapp.de) or not (webnext.integreat-app.de and webnext.malteapp.de)
+    default: false
 
 workflows:
   version: 2
-  commit: # Builds every commit. Delivers to webnext on develop.
+  commit: # Builds every commit. Delivers to webnext.integreat.app and webnext.malteapp.de on develop.
     unless: << pipeline.parameters.api_triggered >>
     jobs:
-      - build:
-          build_config: "integreat-test-cms"
+      - test_and_build
```

Comment:
Not sure about this naming change

Reply:
I think build is enough, as it is not only test but also linting etc.